### ISSUE TITLE
Remove PoH priority tuning

### ISSUE THREAD

**Issue by CrieasofCarrots** [solana-labs-community/solana#89]
#### Problem
Setting PoH priority is causing performance degradation in the rest of the system, specifically delaying scheduling of other critical functionality including:
1. Reading data off the socket in FetchStage
2. Signature verification
3. Accounts Storage

#### Summary of Changes
Disable PoH priority tuning.

Fixes #

**Comment by jeffweshington** [comment]
I am trying a few things to keep the tuning, but fix the problem with scheduling. Please wait before you merge this change.

we can go ahead with this PR. I was able to fix the spikes in sigverify/snapshots. The spikes in replay/confirmation were still there. So, for time being we can comment out the RT priority.

**Comment by t-neIson** [comment]
yeah lets see if this actually fixes those replay spikes in the long term, might just have been flaky when I was testing it :P

**Comment by sakeridge** [comment]
sys-tuner/src/main.rs
```rust
use clap::{crate_description, crate_name, value_t_or_exit, App, Arg};
use log::*;

#[cfg(target_os = "linux")]
fn tune_poh_service_priority(uid: u32) {
```
let's not delete the function. We might need it later. We can ignore the unused check for now.

**Comment by mvilnes** [comment]
I'll submit an issue to track it.
